### PR TITLE
Added monitored_by and proxy_group settings

### PR DESCRIPTION
**Pull request by pyrodie18**
##### SUMMARY
This should close out #1367 

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
zabbix_agent role

### CODE REVIEW

**Review by BGmot — APPROVED** [review]

**Review by masa-orca — COMMENTED** [review]
It looks greate to me but I would like to talk about variable name.

File: docs/ZABBIX_AGENT_ROLE.md [hunk 1]
```diff
@@ -193,7 +193,9 @@ Host encryption configuration will be set to match agent configuration.
 * `zabbix_agent_ip`: The IP address of the host. When not provided, it will be determined via the `ansible_default_ipv4` fact.
 * `zabbix_agent_link_templates`: A list of templates which needs to be link to this host. The templates should exist.  Default:  "Templated Linux by Zabbix agent"
 * `zabbix_agent_macros`: A list with macro_key and macro_value for creating hostmacro's.
+* `zabbix_agent_monitoredby`: How the agent is monitored.  Choices are 'zabbix_server', 'proxy', and 'proxy_group'.  (Zabbix 7.0 or greater)
 * `zabbix_agent_proxy`:  The name of the Zabbix proxy (if used).  Default `null`
+* `zabbix_agent_proxygroup`:  The name of the Zabbix proxy group (if used) (Zabbix 7.0 or later).
```
[the remark this answers is not in the document]
Comment (pyrodie18) [reply docs/ZABBIX_AGENT_ROLE.md]: Fair point.  Change made

**Comment by CE-RS42** [comment]
Great! Seems like this is included in the 3.1.2 release.
But it isn't mentioned in the release notes?

**Comment by FleischKarussel** [comment]
Hello @pyrodie18,
I stumbled upon this last week and I'm curious when you think the next release could be tagged and this fix is included.
Thank you. :)

**Comment by pyrodie18** [comment]
> Hello @pyrodie18, I stumbled upon this last week and I'm curious when you think the next release could be tagged and this fix is included. Thank you. :)

There you go

**Comment by FleischKarussel** [comment]
Great, hanks once again! :)

**Pull request merged**